File: iniciante/javascript/1079.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

let entrada = parseInt(lines.shift())
let countEntradas = 0

while(countEntradas++ < entrada){
    let valores = lines.shift().split(' ')
    let mediaPonderada = (parseFloat(valores[0]) * 2 + parseFloat(valores[1]) * 3 + parseFloat(valores[2].replace('\r', '')) * 5)/10
    console.log(mediaPonderada.toFixed(1))
}

File: iniciante/python/1079.py
# -*- coding: utf-8 -*-

count_entradas = 0

qtd_entradas = int(input())
numeros = []
entrada_numeros = ''
media_ponderada = 0.0
while count_entradas < qtd_entradas:
    entrada_numeros = input().split(' ')
    entrada_numeros = [float(x) for x in entrada_numeros]
    
    media_ponderada = (entrada_numeros[0] * 2 +
                       entrada_numeros[1] * 3 +
                       entrada_numeros[2] * 5) / 10
    
    print(f'{media_ponderada:.1f}')

    count_entradas += 1
